src/ir/function.rs

```diff
@@ -150,7 +150,7 @@ impl FunctionSig {
             CXCursor_CXXMethod => {
                 // For CXCursor_FunctionDecl, cursor.args() is the reliable way
                 // to get parameter names and types.
-                cursor.args()
+                cursor.args().expect("It cannot be None because we are in a method/function")
```

Comment:
Is expect the right way to handle that there?
Can we rely on the fact that `cursor.args()` is now None in case it is not a function or method?